frontend/src/scenes/insights/ActionFilter/ActionFilterRow/ActionFilterRow.tsx

```diff
@@ -333,103 +339,122 @@ export function ActionFilterRow({
-                                                        </div>
-                                                    </Tooltip>
-                                                )}
-                                            />
-                                        </Col>
-                                    </>
-                                )}
-                            </>
-                        )}
-                        {(horizontalUI || fullWidth) && !hideFilter && !readOnly && <Col>{propertyFiltersButton}</Col>}
-                        {(horizontalUI || fullWidth) && !hideRename && !readOnly && <Col>{renameRowButton}</Col>}
-                        {(horizontalUI || fullWidth) && !hideFilter && !singleFilter && !readOnly && (
-                            <Col>{duplicateRowButton}</Col>
-                        )}
-                        {!hideDeleteBtn && !horizontalUI && !singleFilter && !readOnly && (
-                            <Col className="column-delete-btn">{deleteButton}</Col>
-                        )}
-                        {horizontalUI && filterCount > 1 && index < filterCount - 1 && showOr && orLabel}
+                            )}
+                            {showSeriesIndicator && <Col className="action-row-letter">{seriesIndicator}</Col>}
+                        </div>
+                        {/* central section flexible */}
+                        <div className="row-center">
+                            {customRowPrefix !== undefined ? (
+                                <Col>{prefix}</Col>
+                            ) : (
+                                <>{horizontalUI && <Col>Showing</Col>}</>
+                            )}
+                            <Col
+                                className="column-filter"
+                                style={
+                                    fullWidth
+                                        ? {}
+                                        : {
+                                              maxWidth: `calc(${
+                                                  mathAvailability === MathAvailability.None ? '100' : '50'
+                                              }% - 16px)`,
+                                          }
+                                }
+                                flex={fullWidth ? 'auto' : undefined}
+                            >
+                                {filterElement}
+                            </Col>
+                            {customRowSuffix !== undefined && <Col className="column-row-suffix">{suffix}</Col>}
+                            {mathAvailability !== MathAvailability.None && (
+                                <>
+                                    {horizontalUI && <Col>counted by</Col>}
+                                    <Col
+                                    // style={{ maxWidth: `calc(50% - 16px${showSeriesIndicator ? ' - 32px' : ''})` }}
```

Comment:
What's this? 👀

Reply:
Left over from me comparing what it used to be and what it is now. Will delete